sklearn/utils/tests/test_array_api.py

```diff
@@ -351,8 +293,8 @@ def __init__(self, device_name):
     assert array1.device == device(array1, array1, array2)
 
 
-# TODO: add cupy and cupy.array_api to the list of libraries once the
-# the following upstream issue has been fixed:
+# TODO: add cupy to the list of libraries once the the following upstream issue
+# has been fixed:
 # https://github.com/cupy/cupy/issues/8180
```

Comment:
Note to reviewers: this was fixed upstream and will be included in the next release (v14) but we still have to wait.